.github/workflows/pull-request.yaml

```
@@ -42,6 +42,17 @@ jobs:
         with:
           node-version: ${{ vars.NODEJS_VERSION }}
       - run: make lint
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)